src/appleseed/renderer/meta/tests/test_phasefunction.cpp

```diff
@@ -146,19 +141,19 @@ TEST_SUITE(Renderer_Modeling_PhaseFunction)
             SamplingContext sampling_context(rng, SamplingContext::RNGMode);
 
             float integral = 0.0f;
+            sampling_context.split_in_place(2, NumberOfSamples);
```

Comment:
It's better to initialize the sampling context in one go, like I showed in my previous comment.